test/e2e-encryption/encryption_test.go

```diff
@@ -628,3 +638,34 @@ func (p *provider) EncryptedGRs() []schema.GroupResource {
 func (p *provider) ShouldRunEncryptionControllers() (bool, error) {
 	return true, nil
 }
+
+func extractOperatorSpec(obj *unstructured.Unstructured, fieldManager string) (*applyoperatorv1.OperatorSpecApplyConfiguration, error) {
+	castObj := &operatorv1.OpenShiftAPIServer{}
```

Comment:
isn't the test using some fake resource ?

Reply:
correct :man_facepalming:, I blindly copy-pasted these and didn't check